tests/lean/interactive/Diff.lean.expected.out

```diff
@@ -1,20 +1,18 @@
-{ goals := #[{ type := Lean.Widget.TaggedText.tag
-                         { subexprPos := "/", diffStatus? := none }
-                         (Lean.Widget.TaggedText.append
-                           #[Lean.Widget.TaggedText.tag
-                               { subexprPos := "/0", diffStatus? := none }
-                               (Lean.Widget.TaggedText.append
-                                 #[Lean.Widget.TaggedText.tag
-                                     { subexprPos := "/0/0/1", diffStatus? := none }
-                                     (Lean.Widget.TaggedText.text "x"),
-                                   Lean.Widget.TaggedText.text " = ",
-                                   Lean.Widget.TaggedText.tag
-                                     { subexprPos := "/0/1", diffStatus? := none }
-                                     (Lean.Widget.TaggedText.text "x")]),
-                             Lean.Widget.TaggedText.text " → ",
-                             Lean.Widget.TaggedText.tag
-                               { subexprPos := "/1", diffStatus? := none }
-                               (Lean.Widget.TaggedText.text "True")]),
+{ goals := #[{ type := Lean.Widget.TaggedText.append
```

Comment:
@Vtec234 I wanted to flag that this test changed. I modified how the delaborator adds terminfo — it now doesn't add terminfo to syntax that already has it. I haven't really gotten to the bottom of why this test has changed though. If you find it to be worrisome, I'll spend more time trying to figure it out. The infoview seems to have the same diffs at least.

Reply:
I think the top-level expression not being tagged anymore might mean that it can no longer be selected with shift-click. Are you able to select it?

Reply:
Thanks, I see some divergence in hovers. (Strangely, it's that in `∀ (x : Nat), x = x → True`, the `x = x → True` lost its tag.) I'll aim for getting this PR to preserve this test file.